Sources/_StringProcessing/ConsumerInterface.swift

```diff
@@ -441,15 +442,23 @@ private func consumeScalarScript(
   _ s: Unicode.Script
 ) -> MEProgram<String>.ConsumeFunction {
   consumeScalar {
-    Unicode.Script($0) == s
+    if #available(macOS 9999, *) {
+      return unsafeBitCast($0.properties.script, to: Unicode.Script.self) == s
+    } else {
+      return false
+    }
```

Comment:
Can we avoid this somehow? Will `_StringProcessing` always be built with the 5.7 stdlib?

Reply:
> Will _StringProcessing always be built with the 5.7 stdlib?

I believe so

Reply:
Will this availability check be able to go away once you add availability attributes @rxwei ?

Reply:
It's not enough for _StringProcessing to be built with a recent stdlib -- its deployment target must also match the 5.7 release.

To prevent headaches, please use `SwiftStdlib 5.7` instead of explicit 9999 versions whenever possible. Never single out macOS.

Reply:
Yeah the availability check here was just a placeholder for the meantime until availability is figured out. I think don't `SwiftStdlib 5.7` will work in this repo however. I think building this module from build-script will have that macro available, but not when building from SPM.

Reply:
Right, I'm adding availability SwiftStdlib 5.7 to all API. PR coming soon...

Reply:
Need to define these platforms in the package:
```swift
let availabilityDefinition = PackageDescription.SwiftSetting.unsafeFlags([
    "-Xfrontend",
    "-define-availability",
    #""SwiftStdlib 5.7:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999""#
])
```